source/agora/node/BlockStorage.d

```diff
@@ -962,14 +979,17 @@ public class MemBlockStorage : IBlockStorage
 
     /***************************************************************************
 
+        Adds the genesis block to the in-memory blockchain storage.
+
         Returns:
-            true if the storage is empty (height index was not loaded)
+            false as the storage was empty before the genesis block was added
 
     ***************************************************************************/
 
-    public bool isEmpty ()
+    public bool load ()
```

Comment:
It looks like the return value is only used in tests ? I don't think we need this information to be honest, it looks like a leaky abstraction...

Reply:
Like I mentioned before, I need to add the UTXO set from the genesis block in the UTXO PR.

I should only add the UTXOs if the genesis block was added for the very first time.